integrationtests/test_all_examples.py

```diff
@@ -254,16 +280,10 @@ def run_file(file_path):
 class TestExample(unittest.TestCase):
```

Comment:
Nit: Another header before this class would be good so that it's clear where "utility functions" end.